community/blog/2022-10 Arcus Templates v1.0 a major release with major features/README.md

```diff
@@ -0,0 +1,91 @@
+
+## Adding Azure EventHubs project templates
+One of the requested features during an internal Arcus survey was to include EventHubs messaging support. Arcus Messaging v1.3 made this possible, but unlike Service Bus messaging there was not yet any kind of project template that included this type of messaging. Arcus Templates v1.0 comes both with an EventHubs .NET worker template and an Azure Functions EventHubs trigger template (which supports isolated and in-process).
+
+Project templates are a great way to experiment and play with Arcus functionality. They are living code samples that are updated constantly to support the latest features.
+EventHubs messaging is no exception.
+
+We included in these new project templates a useful example of EventHubs messaging. Instead of using `Order` messages like the Service Bus project templates, we opt to create a sample message handler that received sensor readings.
+```csharp
+using Arcus.Messaging.Abstractions;
+using Arcus.Messaging.Abstractions.EventHubs;
+using Arcus.Messaging.Abstractions.EventHubs.MessageHandling;
+using Microsoft.Extensions.Logging;
+
+public class SensorReadingAzureEventHubsMessageHandler : IAzureEventHubsMessageHandler<SensorReading>
+{
+    private readonly ILogger<SensorReadingAzureEventHubsMessageHandler> _logger;
+
+    public SensorReadingAzureEventHubsMessageHandler(ILogger<SensorReadingAzureEventHubsMessageHandler> logger)
+    {
+        _logger = logger;
+    }
+
+public Task ProcessMessageAsync(
+    SensorReading message,
+    AzureEventHubsMessageContext messageContext,
+    MessageCorrelationInfo correlationInfo,
+    CancellationToken cancellationToken)
+    {
+        _logger.LogInformation("Sensor reading processed!");
+
+        return Task.CompletedTask;
+    }
+}
+```
+
+Apart from this, the project's contents is very similar - if not identical: we use the Arcus Messaging EventHubs message pump in the .NET Worker project template and solely register the EventHubs message router in the Azure Functions variant. The message handler, too, is an exact copy in both templates which only shows the reusability of Arcus Messaging.
+For more information on both templates, see our official documentation:
+- [Arcus EventHubs worker project template](https://templates.arcus-azure.net/features/eventhubs-worker-template)
+- [Azure Functions EventHubs trigger project template](https://templates.arcus-azure.net/features/azurefunctions-eventhubs-template)
+
+## Finalizing Web API project template
+The Arcus Web API project template is the oldest and probably the most popular template in our library. It has already been used in several projects and has been improved with feedback to strengthen it for real-life scenarios. The Arcus Templates v1.0 release does not include groundbreaking changes but does however include two small interventions that increase usability.
```

Comment:
```suggestion
The Arcus Web API project template is the oldest and probably the most popular template in our library. It has been used in several projects and has been improved with feedback to strengthen it for real-life scenarios. The Arcus Templates v1.0 release does not include groundbreaking changes but does however include two small interventions that increase usability.
```

Reply:
By the way, not sure if 'interventions' is the right word to use?

Reply:
Maybe we could change it to: `The two small changes in the Web API project template are not groundbreaking but they do increase usability.`

Reply:
Sounds good to me 👍🏻